
<launch>
	
	<node pkg="mal_vehicle" type="map_filter" name="map_filter" output="screen" >

		<param name="filterYamlName" value="$(find mal_vehicle)/cfg/map_filters.yaml"/>
		
		<param name="loadMapName" value="/home/yh/maps_zhenhua/2020-11-04.vtk"/>
		<param name="saveMapName" value="/home/yh/maps_zhenhua/2020-11-04_filtered.vtk"/>
		
		<param name="intTh" value="150"/>
	</node>

</launch>
